pydoctor/test/testpackages/reparenting_follows_aliases/_myotherthing.py

```diff
@@ -0,0 +1 @@
+from ._mything import MyClass
```

Comment:
Can you add a docstring to explain what is the difference between "myotherthing" and "mything"  ? :)

And empty newlines at the end of files :)